### PR TITLE
feat: allow per-page description customization

### PR DESCRIPTION
**Pull request by meysam81**
Hello @Guts 

Thanks for the awesome project.

I use this plugin heavily in my [SRE blog](https://developer-friendly.blog).

I figured it would be nice to allow per page customization of the `description` field, cause at the moment, there's not a lot one can do when the description before `<!-- more -->` is short and extra detail is desired.

With this change, you can have separate contents for RSS description and your typical content.

An example taken from the [Material for Mkdocs Social Cards](https://squidfunk.github.io/mkdocs-material/plugins/social/#option.description):

```markdown
---
date: 2024-06-24
description: >-
  This is the SEO description.
social:
  cards_layout_options:
    description: >-
      This is the social cards description.
rss:
  feed_description: >-
    And I want to have customized RSS description.
---

```

This idea, I reckon, can easily be extended for other similar use-cases:
- #297 
- #275 
- #197

### CODE REVIEW

**Comment by meysam81** [comment]
> can you explain why the `description` is not enough for your need?

`description` is used for SEO description and in order to optimize it, you'd normally keep it short in around 140-156 characters.
RSS description, can go a little wild in the length and doesn't have to comply with anything, especially knowing that it will provide more context to the RSS clients before clicking on `Continue reading...`.


I will add the docs and the tests later this week.

Thanks for the feedback.

**Comment by meysam81** [comment]
@Guts 

Please have another look and let me know if there's any other change required.

**Review by Guts — APPROVED** [review]
LGTM :rocket:!

Thanks a lot for your contribution!

**Comment by Guts** [comment]
Released as part of https://github.com/Guts/mkdocs-rss-plugin/releases/tag/1.15.0

File: docs/configuration.md [hunk 1]
```diff
@@ -173,12 +173,32 @@ Default: `true`.
...
+
+```markdown
+---
+date: 2024-06-24
+description: >-
+  This is the SEO description.
+social:
+  cards_layout_options:
+    description: >-
+      This is the social cards description.
```
[the remark this answers is not in the document]
Comment (Guts) [reply docs/configuration.md:195]: Fair enough. Feel free to make a quick PR to fix it.

**Pull request merged**